packages/graph/hash_graph/lib/graph/src/api/rest/data_type.rs

```diff
@@ -86,7 +87,7 @@ struct CreateDataTypeRequest {
     request_body = CreateDataTypeRequest,
     tag = "DataType",
     responses(
-        (status = 201, content_type = "application/json", description = "The schema of the created data type", body = PersistedOntologyIdentifier),
+        (status = 201, content_type = "application/json", description = "The schema of the created data type", body = PersistedOntologyMetadata),
```

Comment:
Can we change the description of all of the body params we've updated?

Reply:
Good spot, addressed in https://github.com/hashintel/hash/pull/1199/commits/94aead455e8faee24ee583ffe0b006c387147a33